docs/parameters/validation.md

```diff
@@ -114,6 +114,17 @@ params.validationFailUnrecognisedParams = true
         --8<-- "examples/validationFailUnrecognisedParams/pipeline/main.nf"
         ```
 
+## Ignoring unrecognized parameters
+
+Sometimes, a parameter that you want to set may not be described in the pipeline schema for a good reason.
+Maybe it's something you're using in your Nextflow configuration setup for your compute environment,
+or it's a complex parameter that cannot be handled in the schema.
```

Comment:
could you give an example of such complex unrecognized params?

Reply:
I was thinking of nested params. These have their own section, so can link to that.

Reply:
```suggestion
or it's a complex parameter that cannot be handled in the schema, such as [nested parameters](../nextflow_schema/nextflow_schema_specification.md#nested-parameters).
```